ProcessesApi.Tests/V1/Services/ChangeOfNameServiceTests.cs

```diff
@@ -222,14 +224,44 @@ public async Task ProcessStateIsUpdatedToApplicationSubmittedOnSubmitApplication
             // Assert
             CurrentStateShouldContainCorrectData(
                 process, trigger, SharedStates.ApplicationSubmitted,
-                new List<string> { });
+                new List<string> { SharedPermittedTriggers.TenureInvestigation });
 
             process.PreviousStates.Last().State.Should().Be(SharedStates.DocumentChecksPassed);
             VerifyThatProcessUpdatedEventIsTriggered(SharedStates.DocumentChecksPassed, SharedStates.ApplicationSubmitted);
         }
 
         #endregion
 
+        #region Tenure Investigation
+
+        [Theory]
+        [InlineData(SharedValues.Approve, SharedStates.TenureInvestigationPassed)]
+        [InlineData(SharedValues.Decline, SharedStates.TenureInvestigationFailed)]
+        [InlineData(SharedValues.Appointment, SharedStates.TenureInvestigationPassedWithInt)]
+        public async Task ProcessStateIsUpdatedOnTenureInvestigationTrigger(string tenureInvestigationRecommendation, string expectedState)
+        {
```

Comment:
Do we have a test for tenure investigation recommendation invalid?

Reply:
Yes that is added to the ThrowsFormDataNotFoundException tests

Reply:
No, I meant (like in the E2E tests), if there is a value for tenure recommendation but it's not one of approve, decline, appointment etc.

Reply:
That has been added here:
https://github.com/LBHackney-IT/processes-api/pull/113/files/b86b7810515ee0bf95b9fa70a6eb6e9aba129909#diff-ba9efb7dddcf301d44b020329213bb93bddba9fa0325f49d0bf465ab9949ff27R273